v23.1/create-type.md

```diff
@@ -5,13 +5,14 @@ toc: true
 docs_area: reference.sql
```

Comment:
Just noticed that the page summary needs to be updated.

Reply:
Updated!